sup3r/preprocessing/data_handling.py

```diff
@@ -556,6 +556,10 @@ def __init__(self, file_paths, features, target=None, shape=None,
             files do not all have time indices or if there are few input files
             this should be set to one.
         """
+
+        msg = 'No files provided to DataHandler. Aborting.'
+        assert file_paths is not None and file_paths != [], msg
```

Comment:
probably safer to do `assert file_paths is not None and bool(file_paths)`? I dont actually think you need the bool() call here but imo it's easier to read that way.